### ISSUE TITLE
Alternative DRM configurations, are they possible?

### ISSUE THREAD

**Issue by jmartincufre** [kaltura/nginx-vod-module#1170]
Well, as the title says, I want to know if there is any chance to get some of this configurations working:

MSS with Widevine
HLS with Playready (or Widevine)

So far I managed to be able to make it work as expected with the tested configurations. 

Many thanks!

**Comment by erankor** [comment]
No. Afaik, there are 4 different standards for HLS/PlayReady, never heard about MSS/Widevine before...
